### ISSUE TITLE
customize zkRoot path

### ISSUE THREAD

**Issue by xiaoxiangdaiyu** [apache/dubbo-js#238]
The version is "3.0.0-rc6"

In addition to dubbo, our service has other custom roots, such as qa/com.xxxxx.
The default dubbo cannot meet the requirements.

You can see that customization is supported in zookeeper.js
```

//Default dubbo
        this._props.zkRoot = this._props.zkRoot ||'dubbo';
```

But only url is supported in props transmission
```
 register = registry_1.zk({
                url: this._props.register,
            });

```
Expect to add other props, like the following:
```
 register = registry_1.zk({
                url: this._props.register,
                ...this._props
            });
```

**Comment by hufeng** [comment]
@xiaoxiangdaiyu ok, let me implements it.

**Comment by hufeng** [comment]
You can check zookeeper to determine whether the service is registered successfully

**Comment by hufeng** [comment]
landed 4.0.0
